ingestion-data/production/collections/hls-l30-002-ej-reprocessed.json

```diff
@@ -33,10 +33,8 @@
     },
     "license": "MIT",
     "stac_extensions": [
-        "https://stac-extensions.github.io/render/v1.0.0/schema.json",
-        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
```

Comment:
Why do we need to remove the item-assets? 😯

Reply:
I removed it because we didn't define the item_assets for this collection so there is no validation needed for that extension. If we improve this collection to outline all of the included assets on the items we'll need that schema back.